playbooks/common/openshift-node/additional_config.yml

```diff
@@ -0,0 +1,48 @@
+---
+- name: create additional node network plugin groups
+  hosts: "{{ openshift_node_scale_up_group | default('oo_nodes_to_config') }}"
+  tasks:
+  # Creating these node groups will prevent a ton of skipped tasks.
```

Comment:
I'm not sure if that will work.  If it works, that would probably be a better fit here.

Reply:
For Ansible 2.3, the pattern would be to use `static: no` to have properly skip all the tasks in the role.  This would result in only one skipped task, which is the include_role task iteself.

```yaml
  tasks:
  - include_role:
      name: calico
    static: no
    when: openshift_use_calico | default(false) | bool
```
For Ansible 2.4 we can use `import_role` for ‘static’ behaviour and `include_role` for ‘dynamic’.